lxd/container_lxc.go

```diff
@@ -3015,7 +3019,32 @@ func (c *containerLXC) OnStop(target string) error {
 func (c *containerLXC) OnNetworkUp(deviceName string, hostName string) error {
 	device := c.expandedDevices[deviceName]
 	device["host_name"] = hostName
-	return c.setNetworkLimits(device)
+	return c.SetupHostNetworkDevice(device)
+}
+
+// SetupHostNetworkDevice configures a nic device's host side settings.
+func (c *containerLXC) SetupHostNetworkDevice(device types.Device) error {
```

Comment:
This is an internal function (not part of the interface) so should be `setupHostNetworkDevice` instead.